trimmer_decorator.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    trimmed_name = @nameable.correct_name[0..9]
+    trimmed_name.length == 10 ? trimmed_name : @nameable.correct_name
+  end
```

Comment:
- [ ] Let's make sure the name is trimmed whenever the number of character is greater than 10 not when it is equal to 10.

```
def correct_name
    trimmed_name = @nameable.correct_name[0..9]
    trimmed_name.length > 10 ? trimmed_name : @nameable.correct_name
  end
```

Reply:
Oh, I misunderstood the requirement. Thank you so much for pointing that out